schemas/core/booking.json

```diff
@@ -136,7 +136,10 @@
         "MODE_TRAIN": { "$ref": "http://maasglobal.com/core/modes/MODE_TRAIN.json" },
         "MODE_TRAINISH": { "$ref": "http://maasglobal.com/core/modes/MODE_TRAINISH.json" },
         "MODE_BUSISH": { "$ref": "http://maasglobal.com/core/modes/MODE_BUSISH.json" },
-        "MODE_TAXI": { "$ref": "http://maasglobal.com/core/modes/MODE_TAXI.json" }
+        "MODE_TAXI": { "$ref": "http://maasglobal.com/core/modes/MODE_TAXI.json" },
+        "customerNote": {
```

Comment:
Idk if we would ever use customerNote on a global usage. I can see it in TAXIs, and just maybe - train and airline
I suggest you move it to TAXI first, and then later if we need it for any others, we'll reconsider